### PR TITLE
more debugging

### PR DESCRIPTION
**Pull request by maxtaco**
I can't see why exactly CORE-4010 failed, but this will give us more debugging insight into it.

### CODE REVIEW

File: go/engine/identify2_with_uid.go [hunk 1]
```diff
@@ -423,27 +424,32 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
 
 	e.G().Log.Debug("| IdentifyUI.Identify(%s)", e.them.GetName())
 	if err = e.them.IDTable().Identify(e.state, e.arg.ForceRemoteCheck, iui, e); err != nil {
+		e.G().Log.Debug("| Failure in running IDTable")
```
Comment (patrickxb) [inline go/engine/identify2_with_uid.go]: include the error?

File: go/engine/identify2_with_uid.go [hunk 2]
```diff
@@ -423,27 +424,32 @@ func (e *Identify2WithUID) runIdentifyUI(ctx *Context) (err error) {
...
 		return err
 	}
 
 	if waiter != nil {
+		e.G().Log.Debug("+ Waiting for UserCard")
 		if err = <-waiter; err != nil {
+			e.G().Log.Debug("| Failure in showing UserCard")
```
Comment (patrickxb) [inline go/engine/identify2_with_uid.go]: include the error?

**Comment by maxtaco** [comment]
should be caught with the

```go
defer e.G().Log.Debug("- runIdentifyUI(%s) -> %s", e.them.GetName(), libkb.ErrToOk(err))
```

right after the error.

**Comment by patrickxb** [comment]
Ok, sounds good.

**Comment by maxtaco** [comment]
C/I failure I believe is due to this: https://keybase.atlassian.net/browse/CORE-3581

**Pull request merged**